pkgs/by-name/vi/video2x/package.nix

```
@@ -0,0 +1,27 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+
+let
+  pname = "video2x";
+  version = "6.4.0";
+
+  src = fetchurl {
+    url = "https://github.com/k4yt3x/video2x/releases/download/${version}/Video2X-x86_64.AppImage";
+    hash = "sha256-gjj8ODJRB/CgihQNBpuu49uAxax0cLClFsxoMYP1+S0=";
+  };
+
+in
+appimageTools.wrapType2 {
+  inherit pname version src;
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
remove with lib; as it increases eval times